frontend/src/reports/VariableDisparityReport.tsx

```diff
@@ -61,22 +61,54 @@ export function VariableDisparityReport(props: VariableDisparityReportProps) {
     : [];
 
   return (
-    <>
+    <Grid container xs={12} spacing={1} justify="center">
+      {!props.hidePopulationCard && (
+        <Grid item xs={12}>
+          <PopulationCard fips={props.fips} />
+        </Grid>
+      )}
+
       {!variableConfig && (
-        <Grid container xs={12} spacing={1} justify="center">
-          <Grid item xs={5}>
-            <Alert severity="error">Data not currently available</Alert>
-          </Grid>
+        <Grid item xs={5}>
+          <Alert style={{ margin: "20px" }} severity="error">
+            This data is not currently available from our ingested datasets, but
+            will be coming soon.
+            <br />
+            {/* TODO - buttons should be actual working a href links*/}
+            <Button
+              style={{
+                padding: "0",
+                paddingLeft: "5px",
+                paddingRight: "5px",
+                background: "none",
+                textDecoration: "underline",
+              }}
+              onClick={() => alert("unimplemented")}
+            >
+              See our roadmap to learn more.
+            </Button>
+          </Alert>
+          <Alert variant="outlined" severity="info">
+            Do you have information on {props.dropdownVarId} at the state or
+            local level?
+            <Button
```

Comment:
This alert is wider than the one missing data one above it - can they be made the same width?

Reply:
It's in a TODO - there's other UX mocks things that need updating here